src/js/libcs/Operators.js

```diff
@@ -4339,3 +4339,26 @@ evaluator.setsimulationquality$1 = function(args, modifs) {
     }
     return nada;
 };
+
+/**
+ * @todo
+ */
+evaluator.convexhull3d$1 = function(args) {
+    var pointsList = args[0].args[0].stack[0];
```

Comment:
Here is something odd, i would have expected something like evaluateAndVal(args[0]).